### PR TITLE
New section about how people get into network automation

### PR DESCRIPTION
**Pull request by dgarros**
Following up on the proposal that was discussed in Slack, I put together a new section to understand team an individuals are getting into network automation

Very much a draft at this point, please provide your inputs

### CODE REVIEW

File: questionnaire/2019/questions.yaml [hunk 1]
```diff
@@ -217,6 +217,55 @@
...
+  questions:
+  - title: For how long have you been leveraging automation in your network
+    type: Single choice 
+    responses:
+    - Less than 1 year
+    - 1 to 2 years
+    - 2 to 5 years 
+    - More than 5 years
+
+  - title: What actions did your team took to transition to network automation
```
Comment (nickrusso42518) [inline questionnaire/2019/questions.yaml]: grammar; use "take" not "took"

File: questionnaire/2019/questions.yaml [hunk 2]
```diff
@@ -217,6 +217,55 @@
...
+    responses:
+    - Some of us learned on their own
+    - Some of us took some network automation / programming training
+    - We engaged a consulting firm to help get started
+    - We hired a dedicated Network Automation Engineer
+    - We participated in hackathon
+
+  - title: > 
+      If you (personnaly) have transitioned from a network engineer role to a network automation role, 
+      how long did it took you to make the transition
```
Comment (nickrusso42518) [inline questionnaire/2019/questions.yaml]: "take" not "took"

File: questionnaire/2019/questions.yaml [hunk 3]
```diff
@@ -217,6 +217,55 @@
...
+      how long did it took you to make the transition 
+    type: Single Choice 
+    responses:
+    - Less than 1 year
+    - 1 to 2 years
+    - 2 to 5 years 
+    - More than 5 years
+  
+  - title: > 
+      If you (personnaly) have transitioned from a network engineer role to a network automation role,
```
Comment (nickrusso42518) [inline questionnaire/2019/questions.yaml]: "personally"

File: questionnaire/2019/questions.yaml [hunk 4]
```diff
@@ -217,6 +217,55 @@
...
+      If you (personnaly) have transitioned from a network engineer role to a network automation role, 
+      How many hours did you put in training / self learning
+    type: Single Choice 
+    responses:
+    - 100s
+    - 1000s
+    - 5000s
+
+  - title: > 
+      If you (personnaly) have transitioned from a network engineer role to a network automation role,
```
Comment (nickrusso42518) [inline questionnaire/2019/questions.yaml]: "personally"

File: questionnaire/2019/questions.yaml [hunk 5]
```diff
@@ -217,6 +217,55 @@
...
+    - Less than 1 year
+    - 1 to 2 years
+    - 2 to 5 years 
+    - More than 5 years
+  
+  - title: > 
+      If you (personnaly) have transitioned from a network engineer role to a network automation role, 
+      How many hours did you put in training / self learning
+    type: Single Choice 
+    responses:
```
Comment (nickrusso42518) [inline questionnaire/2019/questions.yaml:255]: These answers might be better measured in ranges like our other questions. I also think we should provide a reference like "500 hours is roughly 3 months of full-time work".
```
  - 0 - 500
  - 500 - 2000
  - 2000 - 5000
  - More than 5000
```

**Comment by dgarros** [comment]
Thanks @nickrusso42518 I've made the changes as suggested

**Review by fracaen — APPROVED** [review]
Reviewed/discussed in real time the last changes from dgarros

**Pull request merged**